nanostack-event-loop/eventOS_event_timer.h

```diff
@@ -204,6 +206,17 @@ extern int_fast8_t eventOS_event_timer_request_every(const struct arm_event_s *e
 extern int8_t eventOS_event_timer_cancel(uint8_t event_id, int8_t tasklet_id);
 
 /**
+ * Cancels an event.
+ *
+ * Event can only be a timed event, or user allocated event
+ * that have not yet been processed.
+ *
+ * \param e Pointer to user or timer allocated event or NULL.
```

Comment:
"event". Probably worth being a bit more explicit that cancel(NULL) is a no-op.

Doesn't this work for a normal non-user-allocated event?

Should be declared in eventOS_event.h